inspire_json_merger/conflicts_conf/arxiv_on_arxiv.conf

```diff
@@ -0,0 +1,81 @@
+_collections
```

Comment:
I wouldn't split part of the config off, as this whole package is a config of the json-merger. These files only contain the `filter_out` rules, which has now become less clear.

Reply:
Those lists of field look really ugly to me, that's why I moved them in separate files.